colossalai/fx/profiler/meta_tensor.py

```diff
@@ -0,0 +1,46 @@
+import torch
+from torch.utils._pytree import tree_map, tree_flatten
+
+class MetaTensor(torch.Tensor):
```

Comment:
This might require some refactoring afterwards, either integrate it inside `ColoTensor` or put it directly under `colossalai.fx.tensor`. I can accept as it is now but we can discuss this later.

Reply:
I would consider MetaTensor as an experimental thing. After I develop all possible features, I may refactor this in accordance with its usage and hierarchy.

Reply:
Sure.